applications/nrf_desktop/README.rst

```diff
@@ -62,6 +62,169 @@ The project contains configuration files for following boards:
 The application was designed to allow easy porting to a new HW.
 Check :ref:`porting_guide` for information about the device porting steps.
 
+User interface
+**************
+
+The nRF Desktop devices provide the user input to the host in the same way as other mice and keyboards, using the following options:
+
+    * Connection through USB
+    * Connection through BLE
+
+The devices can also provide input through the nRF Desktop dongle, which is connected to the host through USB.
+The dongle receives data from the peripherals connected through Bluetooth and forwards the data to the host.
+
+The peripheral connection can be based on standard BLE connection parameters or BLE with Low Latency Packet Mode (LLPM).
+LLPM is a proprietary Bluetooth extension from Nordic Semiconductor and can be used only if it is supported by both connected devices (desktop mice do not support it).
```

Comment:
```suggestion
LLPM is a proprietary Bluetooth extension from Nordic Semiconductor.
It can be used only if it is supported by both connected devices - it is supported by gaming mice, but not by desktop mice.
```

Do we have any more information about LLPM somewhere that we could link to?